tests/org.eclipse.ui.tests/Eclipse UI Tests/org/eclipse/ui/internal/ide/ChooseWorkspaceDialogTests.java

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
```

Comment:
Haven't checked, just wondering: don't we need to bump test bundle version too, and don't we need to add the test to the test suite?

Reply:
> Haven't checked, just wondering: don't we need to bump test bundle version too, and don't we need to add the test to the test suite?

The bundle version seems to have been bumped yesterday.

For a test suite, the other tests in the same package are not included into one. But other tests I check are included in a suite... I'll see if I can find a place for the test.